src/main/kotlin/org/jitsi/jibri/Main.kt

```diff
@@ -125,14 +125,24 @@ fun main(args: Array<String>) {
             exitProcess(0)
         }
     }
+    val gracefulShutdownHandler = {
+        logger.info("Jibri has been told to graceful shutdown, waiting to be idle before exiting")
+        jibri.executeWhenIdle {
+            logger.info("Jibri is idle and there are config file changes, exiting")
```

Comment:
`"Jibri is idle and has been told to gracefully shutdown, exiting")`

Reply:
?

Reply:
Got it